AntonioLok/online-shopping-reactJS-Frontend
# update package.json and add docker file

# Changelog
* Add docker to front-end application

# Test
* Build the image by running the command `docker build -t shopping-site/webapp .`
* Check that the image was created successfully using `docker images`
* Run container mapping our port 49180 to the specified in the dockerfile 8080 using`docker run -p 49180:8080 -d shopping-site/webapp`
* Check that the container is running using `docker ps`
* To confirm webapp is running, you can use `docker logs <container-id>` or you can go inside the container using `docker exec -it <container-id>  /bin/sh` and then once you are inside `curl localhost:8080/home`
* Navigate to on the browser `0.0.0.0:49180` or `localhost:49180`
